src/components/modals/SubmitTranscriptModal.tsx

```diff
@@ -1,151 +1,228 @@
   return (
-    <Modal isOpen={isModalOpen} onClose={onClose}>
-      <ModalOverlay />
-      <ModalContent>
-        <ModalHeader>Submitting Transcript</ModalHeader>
-        <ModalCloseButton />
-        <ModalBody>
-          <Box>
-            {steps.map((step, idx) => {
-              const stepLoading = stepIdx === idx && !isError;
-              const stepCompleted = idx < stepIdx;
-              const stepNotRun = idx > stepIdx;
-              const stepError = stepIdx === idx && isError;
-              return (
-                <Flex key={step} alignItems="center" gap={2}>
-                  {stepLoading && (
-                    <Spinner color="orange.300" size="sm" thickness="2px" />
-                  )}
-                  {stepNotRun && (
-                    <Box height="2px" width="14px" bgColor="gray.300"></Box>
-                  )}
-                  {stepCompleted && (
-                    <Icon as={BiCheck} color="green.400" boxSize={6} />
-                  )}
-                  {stepError && <Icon as={BiX} color="red.400" boxSize={6} />}
-                  <Text
-                    fontSize="16px"
-                    color={
-                      stepLoading
-                        ? "orange.400"
-                        : stepCompleted
-                          ? "green.400"
-                          : stepNotRun
-                            ? "gray.300"
-                            : "red.400"
-                    }
-                    fontWeight={stepNotRun ? 300 : 600}
-                  >
-                    {step}
-                  </Text>
-                </Flex>
-              );
-            })}
-          </Box>
-          <Box my={4}>
-            {isError && !isLoading && (
-              <Box>
-                <Text color="gray.400" fontWeight={500}>
-                  Unsuccessful
-                </Text>
-                {err?.message && (
-                  <>
-                    <Divider mt={1} mb={3} />
-                    <Text color="red.400" fontWeight={500}>
-                      Error:
-                    </Text>
-                    <Text color="red.400">{errorMessage}</Text>
-                  </>
-                )}
-              </Box>
+    <>
+      <ModalHeader fontSize="xl">
+        Let&apos;s review your edits
+      </ModalHeader>
+      <ModalBody>
+        <Heading size="sm" pb={2}>
+          Did you make sure the following are accurate?
+        </Heading>
+        <UnorderedList color="gray.700" fontSize="14px">
+          <ListItem>
+            <Text>Title</Text>
+          </ListItem>
+          <ListItem>
+            <Text>Speaker(s)</Text>
+          </ListItem>
+          <ListItem>
+            <Text>Date of original presentation</Text>
+          </ListItem>
+          <ListItem>
+            <Text>Tags for the main topics of discussion</Text>
```

Comment:
I did a grammar check on this and I don't think that your suggestion is correct. This is because "topics" is plural, indicating that there are multiple topics being tagged. Therefore, "main topics" should also be plural to agree with "Tags."

Reply:
Isn't it just one topic or is it multiple topics? If the context here refers to multiple topics, then you're correct. You can discard my suggestion then.

Reply:
multiple topics